src/Transport/Connectivity/MessageReceiverCreator.cs

```diff
@@ -22,12 +24,20 @@ public MessageReceiverCreator(IManageMessagingFactoryLifeCycle factories, ReadOn
             var factory = factories.Get(namespaceAlias);
             var receiveMode = settings.Get<ReceiveMode>(WellKnownConfigurationKeys.Connectivity.MessageReceivers.ReceiveMode);
 
-            var receiver = await factory.CreateMessageReceiver(entityPath, receiveMode).ConfigureAwait(false);
+            var prefetchCount = settings.GetOrDefault<int>(WellKnownConfigurationKeys.Connectivity.MessageReceivers.PrefetchCount);
+            var userHasNotProvidedPrefetchCount = !settings.HasExplicitValue(WellKnownConfigurationKeys.Connectivity.MessageReceivers.PrefetchCount);
+            var transportTransactionMode = settings.HasExplicitValue<TransportTransactionMode>() ? settings.Get<TransportTransactionMode>() : settings.SupportedTransactionMode();
```

Comment:
Update: not needed. Another bug fixed (#344, #345) and now we can accomplish disabling prefetch count (if not specified by user) as part of the default value update during [transport initialization](https://github.com/Particular/NServiceBus.AzureServiceBus/pull/346/commits/945d7c39d0c77976d3d690824af253a54747be63#diff-abc21d576eebe5908bcca158905c4878R73).